packages/core/src/components/Form.js

```diff
@@ -147,7 +147,7 @@ export default class Form extends Component<
     }
   }
 
-  onFieldChange(id: string, value: Value) {
+  onFieldChange(id: string, value: Value, skipFieldTouched) {
     const {
       optionsHandler,
```

Comment:
do i have to provide a type for the `skipFieldTouched` ...like `skipFieldTouched: boolean`

Reply:
Ideally, yes - I'd like to ensure the function signatures are properly typed.